From 61a49d605cefb71d159165839c3d9277998479b5 Mon Sep 17 00:00:00 2001
From: Joel Capitao <jcapitao@redhat.com>
Date: Wed, 28 Jun 2023 09:10:35 +0200
Subject: [PATCH] Remove extra spaces in tox.ini

'setenv' and 'commands' should be at the same level of indentation
otherwise it breaks some build tooling as we are hitting in
RDO.

Change-Id: Ib6ac2e4b62aa26dc9d5c5ff97844f0e7b61386e5
---
 tox.ini | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/tox.ini b/tox.ini
index 5069c6c..45587bb 100644
--- a/tox.ini
+++ b/tox.ini
@@ -21,11 +21,11 @@ commands = {posargs}
 [testenv:cover]
 setenv =
     PYTHON=coverage run --source oslo_concurrency --parallel-mode
-  commands =
-    stestr run {posargs}
-    coverage combine
-    coverage html -d cover
-    coverage xml -o cover/coverage.xml
+commands =
+  stestr run {posargs}
+  coverage combine
+  coverage html -d cover
+  coverage xml -o cover/coverage.xml
 
 [testenv:docs]
 allowlist_externals = rm
-- 
2.40.1

